src/components/Obutton/index.jsx

```diff
@@ -0,0 +1,43 @@
+import React, {Component} from 'react';
+import RaisedButton from 'material-ui/RaisedButton';
+import FlatButton from 'material-ui/FlatButton';
+
+class Obutton extends Component {
```

Comment:
Let's rename the component to Button. We can overlook possible future conflicts right now.